addons/l10n_se/models/res_company.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+import re
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    org_number = fields.Char(compute='_compute_org_number')
+
+    @api.depends('vat')
+    def _compute_org_number(self):
+        for company in self:
```

Comment:
Don't we risk a CacheMiss error here?  We should set company.org_number to '' or False for those